.github/workflows/reusable-check-tag.yml

```diff
@@ -32,6 +32,6 @@ jobs:
           GITHUB_REF="${{ inputs.ref }}"
           TAG=${GITHUB_REF##*/}
           VERSION=$(echo "$TAG" | grep -woP "([0-9]+\.[0-9]+\.[0-9]+)-\w+" | sed -E 's/-(sdk|subgraph|contracts|configs)//')
```

Comment:
Please write a comment what the commands are doing.

Reply:
was already there but improved with the latest commit